### DISCUSSION TITLE
Casa su 4 piani - Consiglio sulla teconologia da utilizzare a aziende

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#3010] [GENERALE - Discussioni generiche]
Ciao a tutti,

abito in una casa su 4 piani (taverna,piano terra, primo piano, e ufficio sottotetto). 

Sto pensando a come renderla smart e nella mia testa vorrei avere:
- termostati intelligienti 
- teste caloriferi intelligneti
- luci smart
- sensori finestre
- toutch panels 
- sensori vari

Ora...Ho già capito che prima di tutto mi servirà un HUB e al momento sto giocando con HA il quale probabilemnte sarà già la mia scelta finale.

Ma per quanto riguarda i sensori/attuatori sono veramente indeciso anche perchè non vorrei avere diversi Gateway come quello della HUE e non vorrei essere legato al cloud (no internet- no controllo) ma dall'altra parte ho 4 piani da coprire e non so bene come fare..

Posso chiedervi dei suggerimenti su cosa voi usereste nel mio caso come prodotti/brand e come gestire i 4 piani?

Grazie e buona Pasqua

**Comment by trebestie** [comment]
In questo momento il massimo della compatibilità è garantito da zigbee.
Se metti un ripetitore, in generale un dispositivo alimentato AC, per piano e prima di comprare qualcosa controlli [qui](https://zigbee.blakadder.com/)  se è supportato e come, non avrai problemi.
